LayoutTests/imported/w3c/web-platform-tests/trusted-types/block-string-assignment-to-DOMWindowTimers-setTimeout-setInterval.html

```diff
@@ -54,7 +54,14 @@
     });
   }, "`window.setInterval(null)` throws.");
 
-  let policy = window.trustedTypes.createPolicy("default", { createScript: x => "0" });
```

Comment:
@ziransun, @lukewarlow, @mrego: FYI those two assertions are never checked because "timeoutTestString"/"intervalTestString" is never passed to an injection sink.

Reply:
Adding a corresponding test in https://phabricator.services.mozilla.com/D232984.